lib/localparticipant.js

```diff
@@ -544,6 +549,7 @@ class LocalParticipant extends Participant {
  * A {@link LocalTrack} was added by the {@link LocalParticipant}.
  * @param {LocalTrack} track - The {@link LocalTrack} that was added
  * @event LocalParticipant#trackAdded
+ * @deprecated
```

Comment:
What do we suggest instead? Do we recommend not using this event at all?

Reply:
@markandrus "trackAdded" and "trackRemoved" in `LocalParticipant` do not serve any purpose, at least in v2.0.